Makefile

```diff
@@ -33,7 +33,7 @@ endif
 include $(ECOLAB_HOME)/include/Makefile
 
 # override the install prefix here
-PREFIX=/usr/local
+PREFIX=/home/wd/usr/minsky#/usr/local
```

Comment:
You can override Makefile variables on the command line:``make PREFIX=~/usr/minsky``

Reply:
I have removed the local prefix, thanks.